controllers/seedimage_controller.go

```diff
@@ -43,14 +43,21 @@ import (
 	"github.com/rancher/elemental-operator/pkg/util"
 )
 
-const seedImgVolSize = 4 * 1024 * 1024 * 1024 // 4 GiB
+const seedImgVolSize = 6 * 1024 * 1024 * 1024 // 6 GiB
 
 type SeedImageReconciler struct {
 	client.Client
 	SeedImageImage           string
 	SeedImageImagePullPolicy corev1.PullPolicy
 }
 
+const (
+	ConfigMapKeyDevice          = "device"
+	ConfigMapKeyRegistrationURL = "registration-url"
+	ConfigMapKeyBaseImage       = "base-image"
+	ConfigMapKeyOutputName      = "output-name"
```

Comment:
Shouldn't these constants be private? My take is that unless they are required by some other package they should be private.

Reply:
I will mark them as private instead!